app/build.gradle

```diff
@@ -92,7 +92,9 @@ android {
 
 	lintOptions {
 		checkReleaseBuilds false
-		disable 'MissingTranslation', 'InvalidPackage', 'VectorDrawableCompat'
+		disable 'MissingTranslation', 'InvalidPackage', 'VectorDrawableCompat',
+				'TrustAllX509TrustManager', 'GradleDependency', 'IconDensities',
+				'PrivateResource', 'Typos', 'TypographyEllipsis'
```

Comment:
Maybe we should add a comment for the non-obvious ones and explain why we disable them? `GradleDependency` and `PrivateResource` for example are not clear to me.

Reply:
`GradleDependency` as we don't want the CI to fail just because of newer available versions of some gradle dependencies. `PrivateResource` is producing a lot of false positives e.g. complaining about R.string.app_name being a private resource of Mapbox. Also it seems some private resources from the google material design library are currently used but I didn't manage to copy them all.
`Typos` and `TypographyEllipsis` may be removed after the next translations fetch as I've corrected these strings in Transifex.

So yes, comments seem to be a good idea here.

Reply:
cool and maybe add a TODO for those that we plan on getting rid of, but can't just yet.

`PrivateResource` sounds like something we should solve eventually.

Reply:
I've added the respective comments.